modules/govuk/manifests/node/s_postgresql_base.pp

```diff
@@ -5,6 +5,7 @@
 class govuk::node::s_postgresql_base inherits govuk::node::s_base {
   include govuk::apps::content_audit_tool::db
   include govuk::apps::content_tagger::db
+  include govuk::apps::ckan::db
```

Comment:
This should be the first item in the list if we're to maintain the alphabetical ordering.